.eslintrc

```diff
@@ -50,7 +50,7 @@
     "key-spacing": [2, { "beforeColon": false, "afterColon": true }],
     "keyword-spacing": [2, { "before": true, "after": true }],
     "linebreak-style": [2, "unix"],
-    "new-cap": 2,
+    # "new-cap": 2, // Temporarily disabled this rule becuase was forcing functions to start with lower case and confusing a React wrapper as a function
```

Comment:
I think we can be a little more specific on this rule. We can adjust it so that using `new` on a constructor requires a capitalized variable name, but a capitalized variable name (like a React component) does not require `new`. [More on this rule](http://eslint.org/docs/rules/new-cap).

```
    "new-cap": [2, { "newIsCap": true, "capIsNew": false }],
```

Further note on temporarily disabling lint rules: we should only change `eslintrc` when we are making an explicit rule change for the entire project. When a rule doesn't apply to a specific use case, it would make sense to disable the rule on a case-by-case basis [where it occurs in the code using inline comments](http://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments). In this case I think we can adjust the rule so that it allows React's higher order components to work anywhere in the project.